feature-libs/product-configurator/rulebased/components/product-title/configurator-product-title.component.html

```diff
@@ -1,92 +1,39 @@
-          <div class="cx-toggle-details-link-text">
-            {{ 'configurator.header.showMore' | cxTranslate }}
-          </div>
-          <cx-icon [type]="iconTypes.CARET_DOWN"></cx-icon>
-        </ng-container>
+    <button (click)="triggerDetails()">
+      <ng-container *ngIf="!showMore">
+        <div class="cx-toggle-details-link-text">
+          {{ 'configurator.header.showMore' | cxTranslate }}
+        </div>
+        <cx-icon [type]="iconTypes.CARET_DOWN"></cx-icon>
+      </ng-container>
 
-        <ng-container *ngIf="showMore">
-          <div class="cx-toggle-details-link-text">
-            {{ 'configurator.header.showLess' | cxTranslate }}
-          </div>
-          <cx-icon [type]="iconTypes.CARET_UP"></cx-icon>
-        </ng-container>
-      </button>
-    </ng-container>
+      <ng-container *ngIf="showMore">
+        <div class="cx-toggle-details-link-text">
+          {{ 'configurator.header.showLess' | cxTranslate }}
+        </div>
+        <cx-icon [type]="iconTypes.CARET_UP"></cx-icon>
+      </ng-container>
+    </button>
     <div class="cx-details" [class.open]="showMore">
       <div class="cx-details-image">
-        <ng-container *cxFeatureLevel="'!3.3'">
-          <img
-            [alt]="getProductImageAlt(product)"
-            [src]="getProductImageURL(product)"
-          />
-        </ng-container>
-        <!-- TODO: update feature level to the release we are publishing with-->
-        <ng-container *cxFeatureLevel="'3.3'">
-          <cx-media
-            [container]="product?.images?.PRIMARY"
-            format="product"
-          ></cx-media>
-        </ng-container>
+        <cx-media
+          [container]="product?.images?.PRIMARY"
+          format="product"
+        ></cx-media>
       </div>
       <div class="cx-details-content">
         <div class="cx-detail-title">
-          <ng-container *cxFeatureLevel="'!3.3'">
-            {{ product.name }}
-          </ng-container>
-          <ng-container *cxFeatureLevel="'3.3'">
-            <span *ngIf="product.name"> {{ product.name }} </span>
-          </ng-container>
+          <span *ngIf="product.name"> {{ product.name }} </span>
         </div>
         <div class="cx-code">
-          <ng-container *cxFeatureLevel="'!3.3'">
-            {{ product.code }}
-          </ng-container>
-          <ng-container *cxFeatureLevel="'3.3'">
-            <span *ngIf="product.code"> {{ product.code }} </span>
-          </ng-container>
+          <span *ngIf="product.code"> {{ product.code }} </span>
         </div>
         <div class="cx-description">
-          <ng-container *cxFeatureLevel="'!3.3'">
-            {{ product.description }}
-          </ng-container>
-          <ng-container *cxFeatureLevel="'3.3'">
-            <span *ngIf="product.description"> {{ product.description }} </span>
-          </ng-container>
+          <span *ngIf="product.description"> {{ product.description }} </span>
```

Comment:
Not related to this PR, but just curious:
Why did we introduce `<span>` wrapper elements here and there in 3.3?

The reason I ask:
For performance of the rendering (especially on mobile): it would be good to avoid DOM elements, unless they are really needed.

Reply:
We realized that, for instance, on the mobile widgets the product title/ description/ code could be longer than expected and that breaks the UI look. After talking to our UI designer, it was decided to truncate the product title / description / code if it does not suite the width of the screen that’s why `span` was added to enable the truncation.